testsuite/integration/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+SKIP_KIND_GENERATION?=""
+
+.PHONY: setup-kind
+setup-kind:
+	@{ \
+		set -e ; \
+		source kind/kind.sh ; \
+		test ! -z "$(SKIP_KIND_GENERATION)" && echo "Skipping cluster generation" || generate_cluster ; \
+		install_dependencies ; \
+		prep_kubeconfig ; \
+	}
+	cd slurm && docker compose build && docker compose up --wait
+	docker network connect slurm_default dws-control-plane
+
+.PHONY: setup-dws
+setup-dws:
+	@{\
+		set -e ; \
+		cd ../submodules/dws ; \
+		docker buildx build -t local/dws-operator:test --cache-from="/tmp/.buildx-cache" --load . ; \
+		IMAGE_TAG_BASE=local/dws-operator VERSION=test KIND_CLUSTER=dws make kind-push deploy ; \
+		kubectl wait deployment -n dws-operator-system dws-operator-controller-manager --for condition=Available=True ; \
```

Comment:
Add `--timeout=60s` to this wait.   In my experience, DWS often needs more than the default 30s.